README.md

```diff
@@ -110,9 +110,13 @@ To verify all the dependencies were successfully installed,
 try to run the tests with the following command:
 
 ```
-yarn test
+yarn test # headless tests
+yarn selenium-test # browser based tests
```

Comment:
The first attempt at running this failed with:

```
The geckodriver executable could not be found on the current PATH. Please download the latest version from https://github.com/mozilla/geckodriver/releases/ and ensure it can be found on your PATH.
```

Please specify that this dependency needs to be installed before running browser based tests. Also separate the two test examples in two different code samples, eg:

```
yarn test

It is also possible to run browser based tests, before trying this please install ..., then run:

yarn browser-test
```

Once installed, the second attempt fails because the data I have in my openwisp radius doesn't match the test.
**We need a way to allow developers to prepare the test env easily, quickly and automatically**.

We could do this with python/shell scripts that are called at the beginning of each browser test.

Eg:

```bash
#!/bin/bash
echo "User.objects.create_user(.....)" | {OPENWISP_RADIUS_PATH}/manage.py shell_plus
```

`OPENWISP_RADIUS_PATH` could be an environment varable.